### PR TITLE
fix: Remove accents in categorization

### PR DESCRIPTION
**Pull request by JF-Cozy**
Accents are removed when creating categories, to regroup contacts for example with "A" and "À" first letter

- Add test for null Indexes

According to https://medium.com/javascript-in-plain-english/not-so-obvious-removal-of-diacritics-in-javascript-explained-and-done-right-52f4aeb3c85 we chose to use `remove-accents`

### CODE REVIEW

File: src/helpers/contactList.spec.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ describe('categorizeContacts', () => {
       { name: 'Xavier', indexes: { byFamilyNameGivenNameEmailCozyUrl: 'X' } },
       { name: 'Zorro', indexes: { byFamilyNameGivenNameEmailCozyUrl: 'Z' } },
       { name: '', indexes: { byFamilyNameGivenNameEmailCozyUrl: '' } },
-      { name: {}, indexes: { byFamilyNameGivenNameEmailCozyUrl: {} } }
+      { name: {}, indexes: { byFamilyNameGivenNameEmailCozyUrl: {} } },
+      { name: 'John', indexes: { byFamilyNameGivenNameEmailCozyUrl: null } }
```
Comment (Crash--) [inline src/helpers/contactList.spec.js:14]: You should add a test with À or È

**Review by Crash-- — APPROVED** [review]
